typings/index.d.ts

```diff
@@ -3354,6 +3357,20 @@ export interface ApplicationCommandChoicesOption extends BaseApplicationCommandO
   choices?: ApplicationCommandOptionChoice[];
 }
 
+export interface ApplicationCommandNumericOptionData extends ApplicationCommandChoicesData {
+  type: CommandOptionNumericResolvableType;
+  minValue: number;
+  min_value: number;
+  maxValue: number;
+  max_value: number;
```

Comment:
why this is dual? because minValue will be same as min_value.

Reply:
https://github.com/discordjs/discord.js/pull/6855/files#diff-86252c6a5caa00d9f8fb12018a8cf70659bc19a5fc2a95f9944c7a2c3e781d96R139-R142

Reply:
I see, thanks.